CMakeLists.txt

```diff
@@ -0,0 +1,70 @@
+cmake_minimum_required(VERSION 3.1)
+
+project(qhotkey VERSION 1.2.2 LANGUAGES CXX)
+
+option(QHOTKEY_EXAMPLES "Build examples" ON)
+
+set(CMAKE_POSITION_INDEPENDENT_CODE ON)
+
+find_package(Qt5 COMPONENTS Core Widgets REQUIRED)
+
+qt5_wrap_cpp(MOC_HEADERS
+    QHotkey/qhotkey.h
+    QHotkey/qhotkey_p.h)
+
+set(LIBS
+    Qt5::Core
+    Qt5::Widgets)
+
+set(SRC_FILES
+    QHotkey/qhotkey.cpp)
+
+if(APPLE)
+    find_library(CARBON_LIBRARY Carbon)
+    mark_as_advanced(CARBON_LIBRARY)
+
+    set(SRC_FILES ${SRC_FILES} QHotkey/qhotkey_mac.cpp)
+    set(LIBS ${LIBS} ${CARBON_LIBRARY})
+elseif(WIN32)
+    set(SRC_FILES ${SRC_FILES} QHotkey/qhotkey_win.cpp)
```

Comment:
`-luser32` propably missing here

Reply:
CMake links with user32 (and a few others e.g kernel32.lib) by default on Windows.